.github/workflows/ci.yml

```
@@ -49,21 +49,40 @@ jobs:
             ~/.cache/sccache
             ~/.cache/uv
             ~/.local/bin
             ~/.local/share/uv
 
       - name: Record Namespace cache state
+        # `cache-hit` is false whenever any single listed path is absent, so
+        # it cannot distinguish an empty volume from a warm one. Record the
+        # restored size of every mounted path alongside it.
         env:
           NAMESPACE_CACHE_HIT: ${{ steps.namespace-cache.outputs.cache-hit }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2088:warning:2:18: Tilde does not expand in quotes. Use $HOME

(shellcheck)